src/vellum/workflows/runner/runner.py

```diff
@@ -938,7 +944,7 @@ def _initiate_workflow_event(self) -> WorkflowExecutionInitiatedEvent:
             span_id=self._initial_state.meta.span_id,
             body=WorkflowExecutionInitiatedBody(
                 workflow_definition=self.workflow.__class__,
-                inputs=self._initial_state.meta.workflow_inputs,
+                inputs=self._initial_state.meta.workflow_inputs or BaseInputs(),
                 initial_state=deepcopy(self._initial_state) if self._should_emit_initial_state else None,
                 trigger=self._trigger.__class__ if self._trigger else None,
```

Comment:
[oos] damn this is interesting. Ideally we have the trigger's attributes appear as `inputs`. I think what that means is that we eventually remove `BaseInputs` in the type definition of `workflow_initiated_event.body.inputs`